curriculum/challenges/english/25-front-end-development/review-basic-html/67072fc183c7ca6c588feb4d.md

```diff
@@ -0,0 +1,59 @@
+---
+id: 67072fc183c7ca6c588feb4d
+title: Basic HTML Review
+challengeType: 24
+dashedName: basic-html-review
+---
+
+# --description--
+
```

Comment:
We might want a small paragraph here like "Review the following concepts below to prepare for the quiz." or something like that

Reply:
✅